service-api/app/src/App/src/Handler/LpasActionsHandler.php

```
@@ -73,8 +85,25 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             }
         }
 
-        //If all criteria pass, request letter with activation key
-        $this->olderLpaService->requestAccessByLetter($lpaMatchResponse['lpa-id'], $lpaMatchResponse['actor-id']);
+        $requestId = null;
+
+        if (($this->featureEnabled)('save_older_lpa_requests')) {
+            $requestId = $this->olderLpaService->storeLPARequest(
+                $lpaMatchResponse['lpa-id'],
+                $userId,
+                $lpaMatchResponse['actor-id']
+            );
+        }
+
+        try {
+            //If all criteria pass, request letter with activation key
+            $this->olderLpaService->requestAccessByLetter($lpaMatchResponse['lpa-id'], $lpaMatchResponse['actor-id']);
+        } catch (ApiException $apiException) {
+            if ($requestId) {
+                $this->olderLpaService->removeLpaRequest($requestId);
```

This fails silently if the record is not there? i.e. feature flag toggled off.